### PR TITLE
Update instructions to build and run the rpc relay

### PR DESCRIPTION
**Pull request by satwikkansal**
Docs were missing couple of steps to build and run the relay server

### CODE REVIEW

File: tutorials/more-tutorials/json-rpc-connections/hedera-json-rpc-relay.md [hunk 1]
```diff
@@ -115,7 +115,9 @@ This effectively means that running and instance of `hedera-json-rpc-relay` on H
 * (7) From the next screen that shows your accounts, copy the value of the "DER-encoded private key" and replace `YOUR_OPERATOR_KEY` in the `.env` file with it.
 * (8) From the same screen, copy the value of "Account ID" and set the value of the `OPERATOR_ID_MAIN` variable in the `.env` file with it.
 * (9) Run `npm install` to install dependencies. Recommended that you have NodeJS version `18` or later for this.
-* (10) Run `npm run start` to start the RPC relay server.
+* (10) Run `npm run setup`. This will link the node_modules to the packages, and the packages together.
+* (11) Run `npm run build`. This will clean and compile the relay library and the server.
```
Comment (bguiz) [inline tutorials/more-tutorials/json-rpc-connections/hedera-json-rpc-relay.md]:
```suggestion
* (10) Run `npm run setup` to link dependencies within its sub-packages.
* (11) Run `npm run build` to build the full project.
```

**Comment by satwikkansal** [comment]
@bguiz I have committed your suggestions :)

**Review by theekrystallee — CHANGES_REQUESTED** [review]
Hi @satwikkansal, thanks for the PR! One last thing before we merge, could you please include a `Signed-off-by` line in the commit message? [Here](https://github.com/hashgraph/hedera-docs/pull/371/checks) are steps to resolve the failed check.

Once the DCO check passes, will approve and merge. Thanks again!

**Comment by satwikkansal** [comment]
@theekrystallee done now!

**Review by theekrystallee — APPROVED** [review]

**Pull request merged**